apps/admin/frontend/src/components/write_in_adjudication_button.tsx

```diff
@@ -146,72 +177,90 @@ export const WriteInAdjudicationButton = forwardRef<HTMLDivElement, Props>(
     ];
 
     return (
-      <Container ref={ref} style={{ zIndex: isFocused ? 10 : 0 }}>
-        <RoundedCheckboxButton
+      <Container
```

Comment:
The diff looks bigger here than it is. `Container` has two new props, `RoundedCheckboxButton` -> `StyledCheckboxButton`. `showSearchSelect` added as now we want to be able to render write-ins without the SearchSelect (when they are in a default state without a vote)